[new-product] Google Nexus (#4332)
captn3m0/endoflife.date: products/google-nexus.md

title: Google Nexus
category: device

> tags: google mobile-phone tablet

Reviewer: "---" are missing at the beginning
```md
---
title: Google Nexus
category: device
tags: google mobile-phone tablet
```

Author: > "---" are missing at the beginning
> 
> ```
> ---
> title: Google Nexus
> category: device
> tags: google mobile-phone tablet
> ```

Haha, thanks.

tags: google mobile-phone tablet
iconSlug: google
permalink: /google-nexus
versionCommand: "Settings -> About Phone -> Regulatory labels"
releasePolicyLink: https://support.google.com/nexus/answer/11227897
releaseColumn: false
releaseDateColumn: true
activeSupportColumn: true
discontinuedColumn: true
eolColumn: Security Updates
customColumns:
-   property: supportedAndroidVersions
    position: after-release-column

>     label: Supported Android # https://en.wikipedia.org/wiki/Google_Nexus

Reviewer 2: "supported android" sounds vague, clarified by adding version:

```suggestion
    # https://en.wikipedia.org/wiki/Google_Nexus
    label: Supported Android Version
```

-   releaseCycle: "6"
    releaseLabel: "Nexus 6"
    releaseDate: 2014-10-15
    support: 2017-10-14
    eol: 2017-10-14
    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_6
    supportedAndroidVersions: 5.0 - 7.1.1 # https://www.gsmarena.com/motorola_nexus_6-6604.php

-   releaseCycle: "Player"
    releaseLabel: "Nexus Player"
    releaseDate: 2014-11-03
    support: 2016-11-02
    eol: 2016-11-02
    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_Player
    supportedAndroidVersions: 5.0 - 8.0 # https://en.wikipedia.org/wiki/Nexus_Player#

-   releaseCycle: "6P"
    releaseLabel: "Nexus 6P"
    releaseDate: 2015-09-29
    support: 2018-09-28
    eol: 2018-09-28
    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_6P
    supportedAndroidVersions: 6.0 - 8.1 # https://www.gsmarena.com/huawei_nexus_6p-7588.php

-   releaseCycle: "5X"
    releaseLabel: "Nexus 5X"
    releaseDate: 2015-09-29
    support: 2017-09-28
    eol: 2017-09-28
    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_5X
    supportedAndroidVersions: 6.0 - 8.1 # https://www.gsmarena.com/lg_nexus_5x-7556.php
---

> 

Reviewer: I think its should be like below, with an empty line above and below the "---"

```md
    link: https://en.wikipedia.org/wiki/Nexus_5X
    supportedAndroidVersions: 6.0 - 8.1 # https://www.gsmarena.com/lg_nexus_5x-7556.php
    
---

```

title: Google Nexus
category: device
tags: google mobile-phone
iconSlug: google

> permalink: /nexus

Reviewer 3: fyi, this permalink is already used: [Nexus Repository OSS](https://endoflife.date/nexus)

Author: Oh, you are right. Should I change it to "google-nexus"?

Reviewer 3: Yes, I think that's a good idea.

permalink: /google-nexus
versionCommand: "Settings -> About Phone -> Regulatory labels"

> releasePolicyLink: https://support.google.com/nexus/answer/4457705

Reviewer 4: ```suggestion
releasePolicyLink: https://support.google.com/nexus/answer/11227897
```

This link is for Google Pixel.

releaseColumn: false
releaseDateColumn: true
activeSupportColumn: true
discontinuedColumn: true

> eolColumn: Security Updates

Reviewer 4: ```suggestion
discontinuedColumn: true
activeSupportColumn: Android updates
eolColumn: Security Updates
```

As suggested in https://github.com/endoflife-date/endoflife.date/pull/4332#discussion_r1468067234.

> activeSupportColumn: true
> discontinuedColumn: true
> eolColumn: Guaranteed Software Updates

Reviewer 4: ```suggestion
discontinuedColumn: true
activeSupportColumn: Android updates
eolColumn: Security Updates
```

Following https://support.google.com/nexus/answer/11227897.

customColumns:
-   property: supportedAndroidVersions
    position: after-release-column
    # https://en.wikipedia.org/wiki/Google_Nexus

>     label: Supported Android Version

Reviewer 4: ```suggestion
    label: Supported Android Versions
```

As this is a versions range.

    link: https://endoflife.date/android

# Supported Android versions range is based on https://www.gsmarena.com/.

> releases:

Reviewer 4: Entries must be ordered by their release date.

Author: I already ordered them by their release date.

Reviewer 4: Yes, but the order must be reversed (from the newest to the oldest). Sorry I should have mentioned that.

> releases:

Reviewer 4: ```suggestion
# Support / EOL dates can be found on https://support.google.com/nexus/answer/11227897.
releases:
```

> -   releaseCycle: "One"

Reviewer 4: ```suggestion
-   releaseCycle: "one"
```

`releaseCycle`s are used in API URLs, they must be lower-case.

    releaseLabel: "Nexus 6P"
    releaseDate: 2015-09-29

>     support: 2018-09-28
>     eol: 2018-09-28

Reviewer 4: ```suggestion
    support: 2017-09-01
    eol: 2018-11-01
```

According to https://support.google.com/nexus/answer/11227897.

- support = No guaranteed Android version updates after
- EOL = No guaranteed security updates after

Used start of the month for both.

Other models must also be updated accordingly.

    eol: 2018-11-01
    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_6P

>     supportedAndroidVersions: 6.0 - 8.1 # https://www.gsmarena.com/huawei_nexus_6p-7588.php

Reviewer 4: ```suggestion
    supportedAndroidVersions: 6.0 - 8.1 # https://en.wikipedia.org/wiki/Nexus_6P
```

https://www.gsmarena.com/huawei_nexus_6p-7588.php says 8.0, not 8.1.

> -   releaseCycle: "S"

Reviewer 4: ```suggestion
-   releaseCycle: "s"
```

`releaseCycle`s are used in API URLs, they must be lower-case.

-   releaseCycle: "5x"
    releaseLabel: "Nexus 5X"
    releaseDate: 2015-09-29
    support: 2017-09-01
    eol: 2018-11-01
    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_5X

>     supportedAndroidVersions: 6.0 - 8.1 # https://www.gsmarena.com/lg_nexus_5x-7556.php

Reviewer 4: ```suggestion
    supportedAndroidVersions: 6.0 - 8.1 # https://en.wikipedia.org/wiki/Nexus_5X
```

https://www.gsmarena.com/lg_nexus_5x-7556.php says 8.0.

> -   releaseCycle: "Q"

Reviewer 4: ```suggestion
-   releaseCycle: "q"
```

`releaseCycle`s are used in API URLs, they must be lower-case.

-   releaseCycle: "player"
    releaseLabel: "Nexus Player"
    releaseDate: 2014-11-03
    support: 2016-11-01

>     eol: 2016-11-01

Reviewer 4: Where can I confirm those dates ? They look wrong consider Nexus Player is upgradable to Android 8 released in 2017.

Reviewer 4: Used march 2018 (found on https://en.wikipedia.org/wiki/Nexus_Player).

    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_Player
    supportedAndroidVersions: 5.0 - 8.0 # https://en.wikipedia.org/wiki/Nexus_Player#

-   releaseCycle: "6"
    releaseLabel: "Nexus 6"

>     releaseDate: 2014-10-15

Reviewer 4: Released in November according to both wikipedia and https://www.gsmarena.com/motorola_nexus_6-6604.php. Where can I confirm this date ?

Reviewer 4: Changed to the 01/11/2014.

    support: 2016-10-01
    eol: 2017-10-01
    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_6
    supportedAndroidVersions: 5.0 - 7.1.1 # https://www.gsmarena.com/motorola_nexus_6-6604.php

-   releaseCycle: "7"
    releaseLabel: "Nexus 7"
    releaseDate: 2013-07-26
    support: 2015-07-01
    eol: 2016-08-01
    discontinued: true

>     link: https://en.wikipedia.org/wiki/Nexus_7_(2013)

Reviewer 4: What about https://en.wikipedia.org/wiki/Nexus_7_(2012) ?

Reviewer 4: Added.

>     supportedAndroidVersions: 4.3 - 5.1 # https://www.gsmarena.com/asus_google_nexus_7_(2013)-5600.php

Reviewer 4: ```suggestion
    supportedAndroidVersions: 4.3 - 6.0 # https://www.gsmarena.com/asus_google_nexus_7_(2013)-5600.php
```

According to both  https://www.gsmarena.com/asus_google_nexus_7_(2013)-5600.php and https://en.wikipedia.org/wiki/Nexus_7_(2013).

-   releaseCycle: "q"
    releaseLabel: "Nexus Q"
    releaseDate: 2012-06-27
    support: 2013-07-01

>     eol: 2013-07-01

Reviewer 4: Where can confirm those dates ?

Reviewer 4: Nexus Q has been removed as it was never officially released (cf. https://en.wikipedia.org/wiki/Nexus_Q).

    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_Q
    supportedAndroidVersions: 4.0 # https://en.wikipedia.org/wiki/Nexus_Q#Hardware_and_software

> -   releaseCycle: "10"

Reviewer 4: What about Nexus 9 ? Should it be added too ?

Reviewer 4: Added

    releaseDate: 2014-10-15
    support: 2017-10-14
    eol: 2017-10-14
    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_6
    supportedAndroidVersions: 5.0 - 7.1.1 # https://www.gsmarena.com/motorola_nexus_6-6604.php

> -   releaseCycle: "Player"

Reviewer 4: ```suggestion
-   releaseCycle: "player"
```

`releaseCycle`s are used in API URLs, they must be lower-case.

    releaseLabel: "Nexus Player"
    releaseDate: 2014-11-03
    support: 2016-11-02
    eol: 2016-11-02
    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_Player
    supportedAndroidVersions: 5.0 - 8.0 # https://en.wikipedia.org/wiki/Nexus_Player#

> -   releaseCycle: "6P"

Reviewer 4: ```suggestion
-   releaseCycle: "6p"
```

`releaseCycle`s are used in API URLs, they must be lower-case.

-   releaseCycle: "s"
    releaseLabel: "Nexus S"
    releaseDate: 2010-12-16
    support: 2012-12-01

>     eol: 2012-12-01

Reviewer 4: Where can I confirm those dates ?

Reviewer 4: Updated as per https://en.wikipedia.org/wiki/Nexus_S#Software.

    link: https://en.wikipedia.org/wiki/Nexus_6P
    supportedAndroidVersions: 6.0 - 8.1 # https://www.gsmarena.com/huawei_nexus_6p-7588.php

> -   releaseCycle: "5X"

Reviewer 4: ```suggestion
-   releaseCycle: "5x"
```

`releaseCycle`s are used in API URLs, they must be lower-case.

-   releaseCycle: "one"
    releaseLabel: "Nexus One"

>     releaseDate: 2010-01-05

Reviewer 4: ```suggestion
    releaseDate: 2010-01-05 # https://en.wikipedia.org/wiki/Nexus_One#History
```

    support: 2012-01-01

>     eol: 2012-01-01

Reviewer 4: Where can I confirm those dates ?

Reviewer 4: Replace with true / false.

    discontinued: true
    link: https://en.wikipedia.org/wiki/Nexus_One
    supportedAndroidVersions: 2.1 - 2.3.6 # https://www.gsmarena.com/htc_google_nexus_one-3067.php

> ---

Reviewer 4: ```suggestion

---
```

> > Nexus is a discontinued line of Android devices by Google and known for providing a pure Android experience.

Reviewer 4: ```suggestion
> Google Nexus is a discontinued line of Android devices designed and supported by Google. They were known for providing a pure Android experience.
```

Small suggestion.

>

> > Nexus devices were succeeded by the Pixel series, which continues the tradition of timely
> > software updates and a seamless Android experience.

Reviewer 4: ```suggestion
Alongside the main smartphone products, the line also included tablet computers and streaming media players; the Nexus line started out in January 2010 and reached its end in October 2016. 

Nexus devices were succeeded by the [Google Pixel series](/pixel), which continues the tradition of timely software updates and a seamless Android experience.
```

Documenting a bit more.

> > It started out in January 2010 and reached its end in October 2016, being replaced by the Google Pixel series.

Reviewer 4: ```suggestion
> Google Nexus is a discontinued line of Android devices (smartphone, tablet computers and streaming media players) designed and supported by Google. They were known for providing a pure Android experience.

The Nexus series started out in January 2010 and reached its end in October 2016, so all Nexus devices are now discontinued and unsupported. The Nexus series were succeeded by the [Google Pixel series](/pixel), which continues the tradition of timely software updates and a seamless Android experience.
```

As suggested in https://github.com/endoflife-date/endoflife.date/pull/4332#discussion_r1468062945.

> > Google Nexus is a discontinued line of Android devices designed and supported by Google. They were known for providing a pure Android experience.
> >
> > Alongside the main smartphone products, the line also included tablet computers and streaming media players.
> > Nexus devices were succeeded by the [Google Pixel series](/pixel), which continues the tradition of timely software updates and a seamless Android experience.

Reviewer 4: ```suggestion
> Google Nexus is a discontinued line of Android devices (smartphone, tablet computers and streaming media players) designed and supported by Google. They were known for providing a pure Android experience.

The Nexus series started out in January 2010 and reached its end in October 2016, so all Nexus devices are now discontinued and unsupported. The Nexus series were succeeded by the [Google Pixel series](/pixel), which continues the tradition of timely software updates and a seamless Android experience.
```

Makes more sense IMO as the first paragraph is only for the "pure" product description located at the top of the page, while the rest of the description is located after the table. Also added a bit of information.
